R/projectInit.R

```diff
@@ -35,17 +37,20 @@ projectInit = function(codeRoot=NULL, dataDir=NULL, outputSubdir=NULL,
 		# This was previously accomplished with project.init2, but that is
 		# not actually necessary with this update.
 		dataDir = codeRoot
-
 	}
 
 	PROJECT.DIR = .selectPath(codeRoot, parent=.niceGetEnv("CODE"),
 								default=getwd())
 	PROCESSED.PROJECT = .selectPath(dataDir, parent=.niceGetEnv("PROCESSED"),
 		default=PROJECT.DIR)
 
+	RAW.PROJECT = .selectPath(dataDir, parent=.niceGetEnv("RAWDATA"),
+		default=PROJECT.DIR)
+
 	# Finalize the options.
 	options(PROJECT.DIR=PROJECT.DIR)
 	options(PROCESSED.PROJECT=PROCESSED.PROJECT)
+	options(RAW.PROJECT=RAW.PROJECT)
 
 	if (!file.exists(PROJECT.DIR)) {
```

Comment:
We're mostly using `getOption` calls rather than the local variables for these (e.g., `PROJECT.DIR`). It looks like this is the only spot where the local is used(?) If so, it'd be a little cleaner to make this also a `getOption` call, then remove the locals altogether and simply have each `.selectPath` call (42-48) inside the corresponding option setting (51-53).